views/main.html

```diff
@@ -28,13 +28,13 @@
                     <a href="#" class="login-register-small"> Sign Up</a>
                 </li>
                 <li>
-                    <a href="#" class="login-register-small">Login</a>
+                    <a href="#" class="login-register-small" data-bs-toggle="modal" data-bs-target="#exampleModal">Login</a>
```

Comment:
I think the id should be specific (like login-form, etc. or whatever is relevant). Using "example" is not the best approach.